src/parser/mod.rs

```diff
@@ -13337,6 +13376,42 @@ impl<'a> Parser<'a> {
         })
     }
 
+    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY)
+    pub fn parse_create_index_expr<OPS: OperatorClass>(
```

Comment:
```suggestion
    /// Parse an expression, optionally followed by ASC or DESC (used in ORDER BY)
    fn parse_create_index_expr<OPS: OperatorClass>(
```
The description for the function sounds incorrect?